src/javaRosa/plugin.js

```diff
@@ -567,9 +567,26 @@ define([
             function itextValidator(property, name) {
                 return function (mug) {
                     var itext = mug.p[property],
-                        hasItext = itext && itext.hasHumanReadableItext();
+                        hasItext = false,
+                        languages = itext.itextModel.languages;
+                    if (itext) {
+                        _.each(['default', 'long', 'short'], function(form) {
+                            if (itext.hasForm(form)) {
+                                hasItext = true;
+                                if (_.find(languages, function(lang) {
+                                    return !itext.get('default', lang);
+                                })) {
+                                    hasItext = false;
+                                }
```

Comment:
I think the 6 lines above this comment can be reduced to
```js
hasItext = !_.find(languages, function(lang) {
    return !itext.get('default', lang);
});
```
However, that seems incorrect because it resets `hasItext` for each value of `['default', 'long', 'short']` where `itext.hasForm(form)` returns true. Seems like it should short circuit the outer loop on the first form with non-blank itext. Should it be this (now growing context to 8 lines above this and changing `_.each` to `_.some`)?
```js
hasItext = _.some(['default', 'long', 'short'], function(form) {
    return itext.hasForm(form) && !_.find(languages, function(lang) {
        return !itext.get('default', lang);
    });
});
```
I'm also in favor of moving this back into a method on the `ItextItem`. I found the name `hasHumanReadableItext` much easier to read than this double-nested conditional loop. Maybe the name needs to be changed?